ui/thread.js

```diff
@@ -155,10 +160,27 @@ module.exports = function () {
             }
 
             this.messages = sort(allMessages)
+
+            this.updateBlockingStatus()
           })
         )
       },
 
+      updateBlockingStatus: function() {
+        var self = this
+        var allAuthors = (this.messages.concat(this.rootMsg)).map((x) => { return x.value.author })
+          .filter((x, index, self) => { return self.indexOf(x) == index })
```

Comment:
Shouldn't need to check for duplicates further down because this line is there to prefilter the list.

Reply:
To clarify, because GitHub selected a lot more lines than the one I intended, I mean this line:

`.filter((x, index, self) => { return self.indexOf(x) == index })`

Reply:
Ah right. Makese sense. Thanks :)